democracy_club/templates/hermes/post_list.html

```diff
@@ -9,12 +9,17 @@
     <div class="ds-stack">
         <h1>Democracy Club Blog</h1>
         <p>Our work, election news, and other items of interest.</p>
-        <ul class="unindented-list">
-            <span aria-hidden="true">🏷️ Topics:</span>
-            {% for tag in all_tags %}
-                <li class="blog-tag"><a href="{% url 'hermes_post_list_by_tag' tag %}">{{ tag }}</a>{% if not forloop.last %}, {% endif %}</li>
-            {% endfor %}
-        </ul>
+        <aside class="ds-filter" aria-labelledby="filter-label">
+            <div class="ds-filter-cluster">
+                <ul>
+                    <li id="filter-label" class="ds-filter-label" aria-hidden="true">Filter:</li>
+                    <li><a href="?all" aria-current="true">All</a></li>
```

Comment:
We're going to need to change `aria-current="true"` depending on the active tag...I expect this looks like adding an `if` statement in the loop and on this line

Reply:
Edited. I think this works but I'm not sure bc of the cache. Is there a better way?

Reply:
Fixed!

Reply:
And pushed to staging